war/src/main/less/base/layout-commons.less

```diff
@@ -69,6 +69,26 @@ body {
   width: 100%;
 }
 
+.app-page-body--one-column {
```

Comment:
something is off with the configure security page, (at least on small screens, i.g. 13" mac pro chrome).

the inputs use the full width to the right, whereas the others are balanced

<img width="1434" alt="image" src="https://user-images.githubusercontent.com/21194782/180857667-29d3190f-c025-42b5-9fb1-15139092b606.png">

Reply:
Does it do that if authorisation isn't set to matrix-based security?

Reply:
it works right if I change it to something else, yes nice find, some css need fixing there?

Reply:
Should be fixed now 👍 🤞